ckanext/subscribe/controller.py

```diff
@@ -70,9 +72,14 @@ def signup(self):
                             'administrator for help'))
             return self._redirect_back_to_subscribe_page_from_request(data_dict)
         else:
-            h.flash_success(
-                _('Subscription requested. Please confirm, by clicking in the '
-                  'link in the email just sent to you'))
+            if dataset_title:
```

Comment:
Since both text is the same it would be better to have one flash message and use a variable to format.
Something like:
```
subscribe_title = dataset_title or group_title
...folder'.format(subscribe_title)
```

Reply:
That's a great idea, just pushed a commit to implement this